core/src/main/java/io/seata/core/rpc/ChannelManager.java

```diff
@@ -43,22 +43,32 @@
 public class ChannelManager {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(ChannelManager.class);
-    private static final ConcurrentMap<Channel, RpcContext> IDENTIFIED_CHANNELS
-        = new ConcurrentHashMap<Channel, RpcContext>();
+
+    private static final ConcurrentMap<Channel, RpcContext> IDENTIFIED_CHANNELS = new ConcurrentHashMap<>();
+
+    /**
+     * the tm client active channel: clientIdentified(applicationId:ip) -> channel
+     * Used to store the active channel of the tm client,
+     * When the tm client reconnects, the value will be updated.
+     */
+    private static final ConcurrentMap<String, Channel> TM_CLIENT_ACTIVE_CHANNELS = new ConcurrentHashMap<>();
```

Comment:
How about remove these two map, and made the IDENTIFIED_CHANNELS correct at
```
io.seata.core.rpc.netty.AbstractRpcRemotingServer.ServerHandler#handleDisconnect
```

Reply:
When the client re-registers, it will update the map value.